src/application/job_manager.py

```
@@ -0,0 +1,41 @@
+from typing import List, Optional, Dict, Any
+from datetime import datetime
+from ..domain.interfaces import JobManager
+from ..domain.models import DownloadJob
+import asyncio
+
+
+class InMemoryJobManager(JobManager):
+    def __init__(self):
```

Ruff (0.12.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)